docs/user-guide/how-to-deploy.rst

```diff
@@ -51,460 +51,13 @@ Docker Compose
 Rasa-Only Deployment with Docker Compose
 ----------------------------------------
 
-It is also possible to deploy a Rasa assistant using Docker Compose without Rasa X.
+It is also possible to deploy a Rasa assistant using Docker-Compose without Rasa X. To do so, you can build your
```

Comment:
@erohmensing can we update the K8s section above here with links to the new Rasa X deployment script?

Reply:
but still leave the other option too :D